content/blog/2020-outreach-partners.md

```diff
@@ -0,0 +1,31 @@
+Title: Bringing PyCon India to a wider audience with your help
+Date: 2020-07-23 13:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-outreach-partners
+
+[**PyCon India**](https://in.pycon.org/2020/) is looking for outreach partners that can help spread the word about the conference in their communities and help bring more fresh faces to the event.
+
+<!-- PELICAN_END_SUMMARY -->
+
+We are the largest gathering of Python users and admirers in India who come together once a year to celebrate the language and share our experience and expertise with the community.

```

Comment:
```suggestion
PyCon India is the largest gathering of Python learners, programmers and enthusiasts in India who come from all over the world together once a year to celebrate the programming language and share their experiences, insight and expertise with the community. Last year alone we had an audience of 3000+, with over 300+ proposals submitted and a packed 2-day agenda in addition to 50+ sponsors supporting us. This year we like to bring forward positive change, and make the conference even more inclusive by bringing forth combined efforts of our communities in helping PyCon India reach more viewers.
```

Reply:
@anistark @pythonindia/pycon-india-2020 Can someone confirm these stats are correct?

Reply:
@anistark :up:

Reply:
I have updated this in the content. If there is something wrong about the stats and need to be changed, we can change it later.

Reply:
Sounds good, I will still ping @anistark again and then resolve the last pending comments.